src/main/java/frontend/AnalyzeTab.java

```diff
@@ -0,0 +1,78 @@
+package frontend;
+
+import utils.MessageConsole;
+
+import javax.swing.*;
+import java.awt.*;
+
+/**
+ * Created by mattpatera on 3/19/17.
+ */
+class AnalyzeTab extends JLabel{
+
+    AnalyzeTab() {
+        this.setLayout(new BorderLayout());
+        this.add(consolePanel(), BorderLayout.PAGE_START);
+        this.add(modelPanel(), BorderLayout.CENTER);
+        this.add(timeRemainingPanel(), BorderLayout.PAGE_END);
+    }
+
+    /**
+     * Console panel holds a live feed of the commands being used by the WEKA
+     * API and what they return.
+     * @return consolePanel
+     */
+    private JPanel consolePanel() {
+        JPanel panel = new JPanel(new BorderLayout());
+
+        JLabel title = new JLabel("Console Output");
+        JTextArea consoleText = new JTextArea("\n" +
+                "\n" +
```

Comment:
Unnecessary amount of lorem ipsum. implementing a feed from the logger would be ideal.